net/mptcp/mptcp_sched.c

```diff
@@ -36,7 +36,7 @@ bool mptcp_is_def_unavailable(struct sock *sk)
 }
 EXPORT_SYMBOL_GPL(mptcp_is_def_unavailable);
 
-static bool mptcp_is_temp_unavailable(struct sock *sk,
+bool mptcp_is_temp_unavailable(struct sock *sk,
 				      const struct sk_buff *skb,
 				      bool zero_wnd_test)
```

Comment:
I think the alignment needs some fixing here, right?

Reply:
I just realized that this change is no longer necessary. Looking again at the BLEST code, then it does not call `mptcp_is_temp_unavailable` directly. I forgot to remove this change when I untwisted the BLEST implementation from the DAPS, OTIAS, and LAMPS schedulers, which I also ported to v0.94.

Beyond that, you are of course right. If we remove the `static` modifier the alignment needs to be corrected. But now I think I'm just reverting that change.